### PR TITLE
Refine wording on coding rules

### PR DESCRIPTION
**Pull request by Zaharid**

### CODE REVIEW

File: doc/sphinx/source/contributing/rules.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-## Rules 
+## Code contributions
 
-**Never commit modifications directly to the master version of the code! Instead, 
-open a new branch of the code and make modifications on that branch. To merge
-changes to master a pull request (PR) should be opened.**
+Code contributions should be presented in the form of [Pull
+Requests](https://github.com/NNPDF/nnpdf/pulls)(PRs) to the repository.
+Never commit modifications directly to the master version of the code. Instead,
+create a new branch and make modifications on it.
```
Comment (scarlehoff) [inline doc/sphinx/source/contributing/rules.md:14]: Maybe we can disable it altogether @Zaharid @scarrazza? (so that only admins can do it)

**Review by scarlehoff — APPROVED** [review]
Other than the comment about the master (I think it is better to be restrictive about it, so maybe rephrase to "pushing to master is disabled") I think this is ok.

File: doc/sphinx/source/contributing/rules.md (hunk 1 above)
Comment (Zaharid) [reply doc/sphinx/source/contributing/rules.md:14]: I think that was the case at some point, but github has a way to toggle stuff unexpectendly for you.

Comment (scarlehoff) [reply doc/sphinx/source/contributing/rules.md:14]: Maybe it's because everyone is an admin? Because we can do: Include administrators Enforce all configured restrictions above for administrators.

Comment (siranipour) [reply doc/sphinx/source/contributing/rules.md:14]: Shouldn't we have it such that to contribute you need to create a fork with the modification?

Comment (scarlehoff) [reply doc/sphinx/source/contributing/rules.md:14]: Well, external people can only do PRs by creating branches.

**Comment by Zaharid** [comment]
Be it as it may, I am going to merge this for now. It is probable good to have that even for people who can push to master.

File: doc/sphinx/source/contributing/rules.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
-## Rules 
+## Code contributions
 
-**Never commit modifications directly to the master version of the code! Instead, 
-open a new branch of the code and make modifications on that branch. To merge
-changes to master a pull request (PR) should be opened.**
+Code contributions should be presented in the form of [Pull
+Requests](https://github.com/NNPDF/nnpdf/pulls)(PRs) to the repository.
+Never commit modifications directly to the master version of the code. Instead,
```
Comment (Zaharid) [inline doc/sphinx/source/contributing/rules.md:13]:
```suggestion
Avoid committing modifications directly to the master version of the code. Instead,
```
